docs/faq.md

```diff
@@ -61,18 +61,18 @@ possible when working with raw webpack configuration data.
 
 ### Can I just re-use my existing webpack configuration?
 
-The Neutrino configuration does let you merge a configuration object, but does not accept a normal webpack configuration
+The Neutrino configuration does let you merge a configuration object but does not accept a normal webpack configuration
 out of the box. Since Neutrino uses webpack-chain and enforces naming of plugins, rules, and loaders, you must use an
 object that corresponds with this "schema". Typically this would involve transforming your webpack configuration to nest
 entities requiring a name into an object which maps the name to the entity.
 
-In short, you must transform your webpack configuration in order to merge it into the Neutrino configuration cleanly.
+In short, you must transform your webpack configuration to merge it into the Neutrino configuration cleanly.
 
 ### What is the difference between middleware and presets?
 
-The term "presets" are a hold-over from Neutrino pre-v5. Presets **are** middleware, the difference in reality is in
+The term "presets" are a hold-over from Neutrino pre-v5. Presets are middleware, the difference, in reality, is in
```

Comment:
The bolding of "are" was intentional here and the previous wording I think was slightly better. Reverting:

```suggestion
The term "presets" are a hold-over from Neutrino pre-v5. Presets **are** middleware, the difference in reality is in
```